secrets/example_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package secrets_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"gocloud.dev/secrets/gcpkms"
+	"google.golang.org/grpc/status"
+)
+
+func ExampleKeeper_ErrorAs() {
```

Comment:
We need an example for using `secrets` overall. I don't think any of our packages have examples for using `As`.

I suggest:

* Dropping this file from the PR.
* Send a separate PR adding this file, but make the example use `localsecrets` and actually work.

I filed a separate PR (#1274) to improve the documentation/examples for `As`, so we can handle the `As` example there.

Reply:
Done.